packages/strapi-connector-bookshelf/lib/queries.js

```diff
@@ -97,14 +106,14 @@ module.exports = function createQueryBuilder({ model, strapi }) {
         : new Date();
     }
 
-    const runCreate = async trx => {
+    const runCreate = wrapErrors(async trx => {
```

Comment:
Sorry for being picky but can you do the same as mongoose :)

```js
 return {
    create: wrapErrors(create),
    update: wrapErrors(update),
}
```

Reply:
np! updated